src/components/ThePostHeader.vue

```diff
@@ -86,79 +105,73 @@
 
 <script>
 import { mapGetters } from 'vuex'
-import { archivePost, reportPost, deletePost } from '@/api'
+import { deletePost } from '@/api'
 import { date } from '@/helper.js'
-import TextEditor from '../components/TheTextEditor'
-import PostStatus from '@/components/PostStatus.vue'
+import LikeButton from '@/components/LikeButton.vue'
+
 export default {
   name: 'the-post-header',
   props: {
     post: { required: true }
   },
   data () {
     return {
-      isArchiving: false,
-      isReporting: false,
       attachments: null
     }
   },
   computed: {
     userPictureUrl () {
-      return this.post.created_by.profile.picture
+      return this.post.created_by && this.post.created_by.profile.picture
     },
-    // @TODO: I18n
     postAuthor () {
       return this.post.created_by && this.post.created_by.profile.nickname
     },
     postCreatedAt () {
       return date(this.post.created_at)
     },
     postUserId () {
-      return this.post.created_by.profile.user_id
+      return this.post.created_by && this.post.created_by.profile.user_id
     },
     ...mapGetters([ 'userId' ])
   },
 
   methods: {
-    async archive () {
-      this.isArchiving = true
-      await archivePost(this.post.id)
-      this.isArchiving = false
-    },
-    async report () {
-      this.isReporting = true
-      await reportPost(this.post.id)
-      this.isReporting = false
-    },
     async deletePost () {
       await deletePost(this.post.id)
       this.$router.push('/')
     }
   },
   components: {
-    TextEditor, PostStatus
+    LikeButton
   }
 }
 </script>
 
 <i18n>
 ko:
   archive: '담아두기'
+  unarchive: '담기 취소'
   edit: '수정'
   delete: '삭제'
   report: '신고'
+  comments: '댓글'
+  views: '조회수'
+
 en:
   archive: 'Archive'
+  unarchive: 'Unarchive'
```

Comment:
🤔 [The opposite of archive](https://english.stackexchange.com/questions/106256/the-opposite-of-archive)